src/documents/views.py

```diff
@@ -910,3 +914,37 @@ def post(self, request, *args, **kwargs):
             return Response({"result": result})
         except Exception:
             return HttpResponseBadRequest()
+
+
+class MailAccountViewSet(ModelViewSet):
+    model = MailAccount
+
+    queryset = MailAccount.objects.all()
```

Comment:
I think this is where the warning can be fixed,  `order_by("pk")` I think?  It might also be able to put in the `class Meta:` for each model to set to more globally.

```
    class Meta:
        ordering = ("name",)
```

Reply:
Ah ok, thanks, I do see others (though not 1) have explicit ordering params. OK let me know if youre still seeing it with the last commit!